### PR TITLE
fix EDGE detection

### PR DESCRIPTION
**Pull request by petercmuc**
## Quick PR Summary
- Updated the condition in Edge detection to fix a bug where not every userAgent was being detected properly.
- Added new Edge userAgent and respective test coverage

## Why
We need to test for "Edge/" AND "Edg/" including the "/". Both terms are "in the wild" in the user-agent strings
https://learn.microsoft.com/en-us/microsoft-edge/web-platform/user-agent-guidance

### CODE REVIEW

**Comment by hollabaq86** [comment]
👋 @petercmuc Thanks so much for the PR! We'll take a look and either provide feedback or merge.

**Comment by hollabaq86** [comment]
(for internal tracking, ticket 1886)

**Comment by petercmuc** [comment]
I'm sorry, but looking at the time it takes to have detection fixed (for Edge - which is a major browser), I doubt that this project is suitable for the changing browser landscape...

**Comment by EnriqueWood** [comment]
@petercmuc It appears that Edge is now being displayed as `Edg`, and this commit effectively resolves the issue even though the pull request has been open for a year. 

Is there a possibility of this being merged soon?

Here's my User Agent (UA):
`Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36 Edg/114.0.1823.51`

File: src/is-edge.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 export = function isEdge(ua?: string): boolean {
   ua = ua || window.navigator.userAgent;
 
-  return ua.indexOf("Edge/") !== -1;
+  return ua.indexOf("Edge/") !== -1 || ua.indexOf("Edg/") !== -1;
```
[the remark this answers is not in the document]
Comment (petercmuc) [reply src/is-edge.ts:4]: We need to test for "Edge/" AND "Edg/" including the "/"...
With regular expression matching we could simplify the expression, though

[the remark this answers is not in the document]
Comment (oscarleonnogales) [reply src/is-edge.ts:4]: Got it. Thanks for putting this PR and fix together! The changes and the unit test look good to me. If you're able to add a CHANGELOG entry I'll go ahead and approve/merge this PR :)

Thanks for your patience as well, I'll be sure to release these changes once the CHANGELOG entry is done. 

```
# Unreleased
 - Fix edge detection by also checking for "Edg/" in userAgent
```

[the remark this answers is not in the document]
Comment (petercmuc) [reply src/is-edge.ts:4]: Done :-)

**Review by oscarleonnogales — APPROVED** [review]
Great work @petercmuc! Thank you for the contribution.

**Review by cgdibble — APPROVED** [review]
